src/uucore/src/lib/features/checksum/validate.rs

```
@@ -758,16 +497,16 @@ fn get_expected_digest_as_hex_string(
 /// Returns a reader that reads from the specified file, or from stdin if `filename_to_check` is "-".
 fn get_file_to_check(
     filename: &OsStr,
-    opts: ChecksumOptions,
+    opts: ChecksumValidateOptions,
 ) -> Result<Box<dyn Read>, LineCheckError> {
     let filename_bytes = os_str_as_bytes(filename).expect("UTF-8 error");
```

ride along: maybe we should propagate the error ?